pkg/systemd/graphs.scss

```diff
@@ -1,27 +1,12 @@
 @import "../lib/plot.css";
-@import "../../node_modules/@patternfly/patternfly/components/Breadcrumb/breadcrumb.css";
-@import "../../node_modules/@patternfly/patternfly/layouts/Gallery/gallery.css";
-@import "../../node_modules/@patternfly/patternfly/components/Page/page.css";
-@import "../../node_modules/@patternfly/patternfly/components/Card/card.css";
```

Comment:
This removal is most likely the cause for the broken graphs page. See PR #13425

Reply:
Hmm, how this slipped in - put back - thanks :)

Reply:
But is that really right? Shouldn't this import the original *.scss files? At least ./node_modules/@patternfly/patternfly/components/Breadcrumb/breadcrumb.scss exists.